src/models/components.d.ts

```diff
@@ -6,6 +6,37 @@ export interface SearchResultItemType {
   image: string;
 }
 
+export interface filterItem {
+  name: string;
+  amount: number;
+}
+
+export interface ageItem {
+  start: number;
+  end: number;
+  unit: string;
+}
+
+export type lifeStageItem = {
+  name: string;
+  icon: SvgIconComponent;
+};
+
+export interface content {
+  nutrient: string;
+  valuePercent: string;
```

Comment:
The type of `valuePercent` should be `number` and not `string`

Reply:
done

Reply:
Ma I have addressed them. Please check very well